### PR TITLE
[CDS-54856]: Adding OCI helm go files manual changes

### PR DESCRIPTION
**Pull request by namantalaycha10**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

File: examples/data-sources/harness_platform_connector_oci_helm/data-source.tf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+data "harness_platform_connector_oic_helm" "example" {
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply examples/data-sources/harness_platform_connector_oci_helm/data-source.tf]: Done

File: examples/resources/harness_platform_connector_oci_helm/resource.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Credential username_password
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply examples/resources/harness_platform_connector_oci_helm/resource.tf]: Done

File: internal/service/platform/connector/oci_helm_test.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+			},
+		},
+	})
+}
+
+func TestAccResourceConnector_ocihelm_UsernamePassword(t *testing.T) {
+
+	id := fmt.Sprintf("%s_%s", t.Name(), utils.RandStringBytes(5))
+	name := id
+	resourceName := "harness_platform_connector_helm.test"
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply internal/service/platform/connector/oci_helm_test.go]: Done

File: internal/service/platform/connector/oci_helm_test.go [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+	"github.com/harness/terraform-provider-harness/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccResourceConnector_ocihelm_Anonymous(t *testing.T) {
+
+	id := fmt.Sprintf("%s_%s", t.Name(), utils.RandStringBytes(5))
+	name := id
+	updatedName := fmt.Sprintf("%s_updated", name)
+	resourceName := "harness_platform_connector_helm.test"
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply internal/service/platform/connector/oci_helm_test.go]: Done

File: internal/service/platform/connector/oci_helm_data_source_test.go [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+		name = "%[1]s"
+		description = "test"
+		tags = ["foo:bar"]
+
+		secret_manager_identifier = "harnessSecretManager"
+		value_type = "Inline"
+		value = "secret"
+	}
+
+		resource "harness_platform_connector_helm" "test" {
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply internal/service/platform/connector/oci_helm_data_source_test.go]: Done

File: internal/service/platform/connector/oci_helm_data_source_test.go [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+	"github.com/harness/harness-go-sdk/harness/utils"
+	"github.com/harness/terraform-provider-harness/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceConnectorOciHelm(t *testing.T) {
+
+	var (
+		name         = fmt.Sprintf("%s_%s", t.Name(), utils.RandStringBytes(4))
+		resourceName = "data.harness_platform_connector_helm.test"
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply internal/service/platform/connector/oci_helm_data_source_test.go]: Done

File: .changelog/458.txt [hunk 7]
```diff
@@ -0,0 +1,6 @@
+```release-note:new-resource
+platform_connector_oci_helm
+```
+```release-note:new-data-source
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply .changelog/458.txt]: Done

File: .changelog/458.txt [hunk 8]
```diff
@@ -0,0 +1,6 @@
+```release-note:new-resource
```
[the remark this answers is not in the document]
Comment (namantalaycha10) [reply .changelog/458.txt]: Done

**Comment by namantalaycha10** [comment]
trigger build

**Comment by namantalaycha10** [comment]
trigger subcategoryfieldcheck

**Comment by namantalaycha10** [comment]
trigger build

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**